ddtrace/constants.py

```diff
@@ -36,3 +37,30 @@
 AUTO_KEEP = 1
 # Use this to explicitly inform the backend that a trace should be kept and stored.
 USER_KEEP = 2
+
+
+_DEPRECATED = {
+    "FILTERS_KEY": "FILTERS",
```

Comment:
maybe a comment here saying that we are removing this entirely as opposed to moving.

it took me a second of reviewing the below TODO comment why this was different.

Reply:
Missed in commit, fixed.